kubectl-cloudflow/cmd/update_docker_registry_credentials.go

```diff
@@ -106,8 +106,9 @@ func createOrUpdateNamespace(k8sClient *kubernetes.Clientset, appID string) {
 func newNamespace(appID string) v1.Namespace {
 	return v1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   appID,
-			Labels: domain.CreateLabels(appID),
+			Name:        appID,
+			Labels:      domain.CreateLabels(appID),
+			Annotations: domain.CreateAnnotations(version.ReleaseTag, version.BuildNumber),
```

Comment:
@olofwalker Idea: we could make a difference between `created-by-cli-version` and `last-modified-by-cli-version` to add even more info.

WDYT?

Reply:
I would also combine the two pieces of information into one value, e.g. `$ReleaseTag ($buildNumber)`, which is consistent with how we show it in the `kubectl cloudflow version` command output.

Reply:
I will see if we can add both those tags, we need to readback the annotations when updating the CR, sounds like a good idea.